admin/app.js

```diff
@@ -248,7 +248,8 @@ app.use(function (req, res, next) {
 const csrf = csurf()
 const csrfExcludedPaths = [
   '/auth', // disable CSRF for NCA tools
-  '/sign-in' // disable CSRF for login
+  '/sign-in', // disable CSRF for login
+  '/test-developer/view-pupil-payload'
```

Comment:
Why is this excluded from CSRF? Do we expect it to need to be called from a script in any way, or can we just add a csrf token to the form? I've seen that the request is returned as raw json, but it's a bit odd that this is POST and comes from the UI, maybe getting it from the query like `/raw-pupil-payload/:checkCode` would be better? 🤔

Reply:
@danwilliams - Good question

The implementation returns the raw json in a new window, leaving the original form.  By omitting the csrf token we can allow the user to open several documents, which seems a reasonable use-case to me when investigating many checks from the reports.

Using GET may work too, will have a look next week.

Reply:
@danwilliams - updated to use GET and the csrf exclusion has been removed.